### ISSUE TITLE
[feat] Top-bar should be available even when chart is on fullscreen

### ISSUE THREAD

**Issue by hugovalente-pm** [netdata/netdata-cloud#389]
## Goal

We need to have the top bar be visible when a chart is seen in fullscreen because user may need an access to the time selector
The option to chose can be whatever is easier for FE:

### Option 1 - just top-bar
![image](https://user-images.githubusercontent.com/82235632/162174726-7919e7e5-84ae-4d40-a268-70e7f419e4bc.png)

### Option 2 - top-bar and left-bar
![image](https://user-images.githubusercontent.com/82235632/162175665-7510e6ae-b2f0-44b2-ac3e-8b011eecc41f.png)

**Comment by vinnygats** [comment]
Basically yes, the topbar and leftbar have a lot of options that do not really affect the chart at all, so not sure if it makes sense to have them exposed really. We may have users that really want to have this chart running on big screens continuously and nothing else.
For me would make more sense to enable the datetime picker within the full screen of the chart.

**Comment by christophidesp** [comment]
I agree that we should always show the top bar in order to maintain control over time selection and war room visibility, so that the context is always clear for the user. As for the side bar, I don't think it is a critical part of the full screen chart layout, but I think it looks more consistent if we keep it visible together with the top bar.

**Comment by hugovalente-pm** [comment]
I agree with @vinnygats comment, about user in the context of a maximized chart not needing to access other things besides datetime picker but we may add new things that could make sense, e.g. we have the Universal node filtering coming up, so I'm more inclined towards keeping the top bar and not moving stuff inside the chart.

I included the left bar on the suggestion because of what @christophidesp mentions, looks more consistent... but would be ok if we were to exclude it because of what I mentioned above.
